README.md

```diff
@@ -1,13 +1,4 @@
-# git-training
-This is a temporary repository
-
-Meine erste Änderung
-
-Meine zweite Änderung
-
-Dies ist ein Test, den ich hinzufügen möchte
-
-Neue Zeile
+Jurij
```

Comment:
are you sure you wanna delete all the above 10 lines?

Reply:
Maybe linke 1 and 2 can stay. The others could be removed from my pov. But it's also fine if they stay. 😄